Source/JavaScriptCore/Scripts/jsmin.py

```diff
@@ -1,3 +1,5 @@
+# vim: set fileencoding=utf-8 :
+
 # This code is original from jsmin by Douglas Crockford, it was translated to
```

Comment:
No license update?

Reply:
No, there were just a couple white space changes, that I removed to reduce noise in the diff.